plugins/modules/openssh_cert.py

```diff
@@ -36,6 +36,18 @@
             - Should the certificate be regenerated even if it already exists and is valid.
         type: bool
         default: false
+    idempotency:
+        description:
+            - When C(partial) and I(force=false) an existing certificate will be regenerated based on
+              I(serial), I(type), I(valid_from), I(valid_to), I(valid_at), and I(principals).
+            - When C(full) and I(force=false), I(identifier), I(options), I(public_key), and I(signing_key)
+              are also considered when compared against an existing certificate.
+        type: str
+        choices:
+            - partial
+            - full
```

Comment:
How about adding an `always_regenerate` option, and document `force` as "will set `idempotency` to `always_regenerate`"?

But maybe it's better to rename this option to `regenerate` so it can be more similar to the option of the same name of openssl_privatekey. Have you seen that option?

Reply:
Yes, I had to work with that pattern for `openssh_keypair` which has a similar implementation. Adding `never` and `always` would be trivial so I can implement the same here.